src/platform/novo-elements.module.ts

```diff
@@ -107,6 +107,7 @@ import { NovoCommonModule } from './elements/common/common.module';
     UnlessModule,
     NovoCommonModule,
     NovoStepperModule,
+    ScrollDispatchModule,
```

Comment:
Err I mean export the ScrollDispatchModule here, not in the demo module... silly me